l10n_it_withholding_tax/models/account.py

```diff
@@ -420,7 +420,7 @@ def create(self, vals):
     def _onchange_invoice_line_wt_ids(self):
         self.ensure_one()
         wt_taxes_grouped = self.get_wt_taxes_values()
-        wt_tax_lines = []
+        wt_tax_lines = [(5, 0)]
```

Comment:
@robyf70 come mai solo uno zero?

Reply:
@eLBati Ho visto che viene spesso usato anche in questa forma, l'operazione 5 ha gli altri 2 argomenti opzionali ed a quanto pare odoo lo accetta senza problemi.

Reply:
Mi sembrerebbe più chiaro `[(5,)]` o `[(5, 0, 0)]`. Comunque OK